en/docs/learn/enterprise-integration-patterns/messaging-channels/message-bus.md

```diff
@@ -1,35 +1,118 @@
 # Message Bus
 
-This section explains how the Message Bus EIP can be implemented using the ESB profile of WSO2 EI. The following topics are covered:
-
-* Introduction to Message Bus
-* How the ESB profile implements the EIP
+This page explains how you can implement a sample scenario of Message Bus EIP using WSO2 Micro Integrator.
 
 ## Introduction to Message Bus
 
-The Message Bus EIP enables separate applications to work together in a decoupled manner so that applications can be easily added or removed without affecting each other. This approach makes maintenance and testing smoother, since editing or removing an application will not affect the functionality of any other application. For more information, refer to http://www.eaipatterns.com/MessageBus.html.
+The Message Bus EIP enables separate applications to work together in a decoupled manner so that applications can be easily added or removed without affecting each other. This approach makes maintenance and testing smoother since editing or removing an application will not affect the functionality of any other application. 
+
+!!! info 
+    For more information, refer to the [Message Bus](http://www.eaipatterns.com/MessageBus.html) documentation.
 
 ![Message bus solution]({{base_path}}/assets/img/learn/enterprise-integration-patterns/messaging-channels/message-bus-solution.gif)
 
-### How the ESB profile implements the EIP
+## Sample scenario
+
+It illustrates how application logic is layered, and how each component of the application logic is separated as a mediator, allowing message processing to be executed in a decoupled manner. The mediation process is explained in the Mediating Messages section of the WSO2 MI documentation.
```

Comment:
Let's point to this section, rather than sayinh WSO2 MI documentation, as we are already in MI docs.